src/lib/db-listeners/index.ts

```diff
@@ -180,6 +180,19 @@ export class DBListeners extends BaseListener {
     this.unmonitorDocumentModel(document);
   }
 
+  public syncDocumentProperties = (document: DocumentModelType, path: string) => {
+    const { user } = this.db.stores;
+    const { key, properties } = document;
+
+    const updatePath = path || this.db.firebase.getUserDocumentPath(user, key, document.uid);
+    const updateRef = this.db.firebase.ref(updatePath);
+    onSnapshot(properties, (newProperties) => {
+      updateRef.update({
+        properties: JSON.stringify(newProperties)
```

Comment:
I don't think we're stringifying properties for other documents. Is there a particular reason to do so here? If so, is there a reason to do so elsewhere as well? It seems like we should be consistent.